### PR TITLE
🐛 Is823/Fixes password reset

### PR DESCRIPTION
**Pull request by pcrespov**
## What do these changes do?

-  🐛  Fixes password reset: In `` validate_confirmation_and_redirect``, we forgot to unmask confirmation code when returned since it was stored in a ``SecretStr``
- ♻️  some renaming

## Related issue/s
- Fixes ITISFoundation/osparc-issues#823

## How to test

- reproduce ITISFoundation/osparc-issues#823

## Checklist

- [x] Unit tests for the changes exist
- [x] Runs in the swarm: test locally
- [x] Documentation reflects the changes

### CODE REVIEW

File: services/web/server/src/simcore_service_webserver/login/handlers_confirmation.py [hunk 1]
```diff
@@ -93,7 +93,7 @@ async def email_confirmation(request: web.Request):
                 # the browser does NOT reloads page
                 #
                 redirect_to_login_url = redirect_to_login_url.with_fragment(
-                    f"reset-password?code={path_params.code}"
+                    f"reset-password?code={path_params.code.get_secret_value()}"
```
Comment (GitHK) [inline services/web/server/src/simcore_service_webserver/login/handlers_confirmation.py:96]: 👍

**Review by GitHK — APPROVED** [review]
I guess this is no longer WIP

**Review by elisabettai — APPROVED** [review]
Thanks for the quick fix! Once merged, this should be hot fixed to production I guess.

**Comment by pcrespov** [comment]
> Thanks for the quick fix! Once merged, this should be hot fixed to production I guess.

@elisabettai we plan to release to production i the coming days ... will make sure it is in staging first

**Pull request merged**